pkg/controller/import-controller.go

```diff
@@ -306,6 +312,9 @@ func (r *ImportReconciler) reconcilePvc(pvc *corev1.PersistentVolumeClaim, log l
 	} else {
 		if pvc.DeletionTimestamp != nil {
 			log.V(1).Info("PVC being terminated, delete pods", "pod.Name", pod.Name)
+			if getSource(pvc) == SourceVDDK {
+				r.saveVddkAnnotations(pvc, pod, log)
```

Comment:
I wanted to save the annotations just before every pod deletion, since that's when the VDDK data is available. So I think this makes sense because there's a pod deletion immediately after this snippet, even if the PVC annotations aren't going to last very long in this code path. It might be more reliable to update the DV straight from the Pod, but I couldn't find a clean way to do that.

Reply:
Well I would expect the DV to also be deleted, otherwise the DV controller will create a new PVC. I guess it doesn't do any harm, but I don't see it being useful.

Reply:
I took this out anyway. This is what was causing the errors in my comment above.